boltdesignsystem/bolt
# feat: Add suggested bolt.rc file for external builds

## Jira

http://vjira2:8080/browse/BDS-1874

## Summary

Adds suggested .boltrc for use in external Bolt builds

## Details

Although http://vjira2:8080/browse/BDS-1874 mentions a starter package.json, since that task was created that part has become unnecessary (thanks @christophersmith262).  I believe that the automated test mentioned in that task also isn't really relevant anymore since the package.json integrity is the main thing it would test.  The docs site inherits from this rc file, so the main thing that would go wrong is if someone adds a component to the wrong file (i.e. the one for the docs site and not the starter).  Not sure how we'd write a test for that 🤔 .

I'm open to suggestions on file naming-- I ended up following the suggestions in the ticket since it seemed most simple, especially since we don't need a package.json file.

## How to test
- Confirm no regressions in the docs site
- Confirm that all components listed in `.boltrc.suggested.js` are relevant to external builds, and that all those components included in the rc file for recent bolt release builds are present.
